website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -131,7 +131,7 @@ In the `publish_release` CI workflow, the `build_hermes_macos` step produces a `
 
 ### 9. Create a tracking discussion post
 
-Create a "Road to <YOUR_MINOR_VERSION>" discussion post in the [`react-native-releases`](https://github.com/reactwg/react-native-releases/discussions) working group:
+Create a `Road to <YOUR_MINOR_VERSION>` discussion post in the [`react-native-releases`](https://github.com/reactwg/react-native-releases/discussions) working group:
```

Comment:
is this change ok? We can find another alternative like 

```
"Road to `<YOUR_MINOR_VERSION>`"
```

Reply:
Will using square brackets instead of lt/gt characters fix the issue? (Create a "Road to [YOUR_MINOR_VERSION]" discussion post in the...)

Reply:
Yes! Pushing a fix

<img width="832" alt="CleanShot 2023-08-03 at 19 43 34@2x" src="https://github.com/facebook/react-native-website/assets/749374/abbf90e5-25c5-406e-891d-e6cb9e9a8a2b">